datetime_tz/win32tz_map.py

```diff
@@ -0,0 +1,110 @@
+'''Map between Windows and Olson timezones taken from http://www.unicode.org/repos/cldr/trunk/common/supplemental/windowsZones.xml
```

Comment:
We shouldn't be committing this file to the repo right? Your code below generates it as part of the sdist step?